CommonMark/CommonMarkConverter.cs

```diff
@@ -13,26 +13,24 @@ namespace CommonMark
     /// </summary>
     public static class CommonMarkConverter
     {
-        private static Version _version = new Version(0, 0);
+        private static Lazy<Assembly> _assembly = new Lazy<Assembly>(InitializeAssembly, LazyThreadSafetyMode.None);
```

Comment:
I was trying to reduce code duplication, and Lazy<> was readily implemented. I don't hold this change dear.